examples/pom.xml

```diff
@@ -50,6 +50,17 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client</artifactId>
+      <version>3.2.1</version>
```

Comment:
is there a reason why this must be `3.2.1`? note that the there is a pom.xml global property for `${hadoop.version}` which is currently set to `3.3.1`, defined in the root pom.xml. a hardcoded version may eventually drift out of date

Reply:
No specific reason to 3.2.1 version. We can use the latest version of Hadoop version, or Alluxio tested ones.

The initial purpose is to make these examples to be standalone.
These examples can be used to show how to programming with Alluxio released packages.
They can be compiled with Alluxio source code.

Reply:
my understanding is that you are changing the examples module to be standalone, but i see that the modifications put the pom.xml in an awkward situation.
previously, the hadoop dependency versions and alluxio module dependencies are all depending on the root pom.xml. this may not be the best example to show the user.

to make the examples module standalone, it would need to define it's own versions for hadoop and alluxio dependencies. this was done for hadoop, but the current commit does not do it for the alluxio dependencies. this seems like a halfway state. i think my recommendation is if we choose to go the standalone route, then we should also hardcode the alluxio version and comment out to show how to use locally compiled dependencies.

so i would summarize the choice to be between:
- leave as not standalone -> all dependencies are linked to or using the same version as the locally compiled ones from the project = current state before this PR
- change to standalone -> all dependencies are unlinked and are statically defined = take this PR and switch the commented section with the uncommented section for alluxio dependencies

what do you think?

Reply:
I prefer to make these examples as standalone.
These examples are going to show how a developer can write a Java program to do file system operations with Alluxio (with Alluxio native API or Hadoop compatible File System). So, developers can integrate such the Alluxio access into their own applications or compete frameworks.

Which directory should these examples go to?
Maybe we don't want to mix them with the existing ones.